ansible/site.yml

```diff
@@ -0,0 +1,82 @@
+---
+- hosts: primary
+  gather_facts: no
+
+  tasks:
+  - name: install python 2
+    raw: test -e /usr/bin/python || (apt -y update && apt install -y python2.7)
+    become: yes
+
+- name: Provision ISIC Archive
+  hosts: primary
+
+  vars:
+    mongo_data_path: "/srv/mongodb"
+    bind_node_modules: false
+    ansible_python_interpreter: /usr/bin/python2.7
+
+  pre_tasks:
+    - name: Find path to storage volume
+      shell: |
+        readlink -f $(ls /dev/disk/by-id/nvme-Amazon_Elastic_Block_Store_{{ storage_volume_id }})
```

Comment:
Could you mount this block device directly?

Reply:
You may be able to use a "tag" syntax from the [mount command]http://man7.org/linux/man-pages/man8/mount.8.html), but possibly not for `ID`.